src/ci.ml

```diff
@@ -106,7 +106,8 @@ let can_build =
   | `Production ->
     any [
       username "admin";
-      username "github:samoht";
```

Comment:
😢

Reply:
Oh. Do you want to remain an admin of LinuxKit-CI? I thought I'd just forgotten to remove you when you left Docker.